src/compas_timber/connections/t_butt.py

```diff
@@ -1,12 +1,13 @@
-from compas_timber.connections.butt_joint import ButtJoint
-from compas_timber.elements import CutFeature
-from compas_timber.elements import MillVolume
+from compas_timber._fabrication import JackRafterCut
+from compas_timber._fabrication import Lap
+from compas_timber.connections.utilities import beam_ref_side_incidence
 
 from .joint import BeamJoinningError
+from .joint import Joint
 from .solver import JointTopology
 
 
-class TButtJoint(ButtJoint):
+class TButtJoint(Joint):
```

Comment:
so `ButtJoint` is  not used anymore, right? 
if so, perhaps remove it?

Reply:
The `ButtJoint` class was used primarily for common geometric operations like generating the boolean subtraction solid. If I understand correctly, the geometric features are generated by the `btlx_process`es, so `ButtJoint` can presumably be phased out.